src/main/java/de/tum/cit/aet/artemis/core/api/AbstractApi.java

```diff
@@ -0,0 +1,4 @@
+package de.tum.cit.aet.artemis.core.api;
+
+public interface AbstractApi {
```

Comment:
What are you planning to add in this interface later on?

Reply:
I see your point. Discussed that with Stephan - idea is to leave that option for now as changing it afterwards might be more of a pain.